ekf2: fix sideslip timeout (typo)

dagar opened the pull request:
- regression from https://github.com/PX4/PX4-Autopilot/pull/20364

dagar commented:
@bresch is resetting the airspeed timeout in controlBetaFusion() here intentional? 
https://github.com/PX4/PX4-Autopilot/blob/75c63aee2a1d221fbd5a1c5e5ca8b4280fddefc0/src/modules/ekf2/EKF/control.cpp#L736-L743

bresch commented:
Thanks for finding this.
> @bresch is resetting the airspeed timeout in controlBetaFusion() here intentional?

no, it should also be `_aid_src_sideslip`

dagar commented:
> Thanks for finding this.
> 
> > @bresch is resetting the airspeed timeout in controlBetaFusion() here intentional?
> 
> no, it should also be `_aid_src_sideslip`

Fixed and squashed into the same commit.

bresch reviewed (approved):
Thanks
